index.html

```diff
@@ -2324,6 +2324,52 @@ <h3>Presentations</h3>
 [[?VC-JOSE-COSE]] specification.
         </p>
 
+        <section>
+          <h4>Enveloped Verifiable Credentials</h4>
+
+          <p>
+It is possible to include one or more <a>verifiable credentials</a> in a
+<a>verifiable presentation</a> that have been secured using a securing mechanism
+that "envelopes" the payload, such as [[[?VC-JOSE-COSE]]] [[?VC-JOSE-COSE]].
```

Comment:
```suggestion
that "envelopes" the payload, such as [[[VC-JOSE-COSE]]] [[VC-JOSE-COSE]].
```

The reference to VC-JOSE-COSE is normative elsewhere in the document, including in this section, to putting it here as an informative reference does not have any effect. It would just mess up other editors now or later...

Reply:
It's cited here as an informative. Throughout the entire document, I try to be very careful about when we cite something normatively vs. informatively. This is important when sections are added/removed, etc. Over time, some citations that were normative become informative. On that basis, I am going to not take this change.

Reply:
As I said, this is not a show-stopper.